src/contact/contact.service.ts

```diff
@@ -0,0 +1,30 @@
+import { ResultSetHeader } from 'mysql2';
+import { v4 as uuidv4 } from 'uuid';
+import { addedContact201, internalServerError500, updatedContact200 } from '../constants';
+import { WorkWithDB } from '../interfaces';
+import { sequelize } from '../servers';
+import { showResult, updateStringHelper } from './contact.utils';
+
+export const creatNewContact: WorkWithDB = async (req) => {
+  const { name, email } = JSON.parse(req.body.contact);
+  const query = `INSERT INTO contacts (uuid, name, email, createdAt, updatedAt) VALUES('${uuidv4()}', '${name}', '${email}', now(), now());`;
+  console.log(query);
```

Comment:
remove unnecessary console.log

Reply:
Removed unnecessary console.log.
Checked every file in src/ for redundant console.log's.